content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -46,18 +46,34 @@ The number '0' in the 'RESTARTS' column means liveness probes worked fine. Readi
 
 ### Mutual TLS enabled
 
-Run this command to enable mutual TLS for services in the default namespace:
+Run these commands to enable mutual TLS for services in the default namespace. Note that you need both authentication policy and
```

Comment:
Consider using a numbered list for procedures. Having multiple commands and steps in a single paragraph makes translation harder, adds complexity to the content, and makes maintaining the content, updating it and correcting it, a lot harder. Consider these edits:
To enable mutual TLS for services in the default namespace, you must configure an authentication policy and a destination rule. Follow these steps to complete the configuration:

1. To configure the authentication policy, run:
    {{< text bash >}}
    $ cat <<EOF | istioctl create -f -
     apiVersion: "authentication.istio.io/v1alpha1"
     kind: "Policy"
     metadata:
       name: "default"
       namespace: "default"
     spec:
       peers:
       - mtls: {}
    EOF
    {{< /text >}}

1. To configure the destination rule, run:

    {{< text bash >}}
    $ cat <<EOF | istioctl create -f -
    apiVersion: "networking.istio.io/v1alpha3"
    kind: "DestinationRule"
    metadata:
      name: "default"
      namespace: "default"
    spec:
      host: "*.default.svc.cluster.local"
      trafficPolicy:
        tls:
          mode: ISTIO_MUTUAL
    EOF
    {{< /text >}}

Remember to indent the code blocks with four spaces for the list to work properly.

Reply:
Done